dw-surface.js

```diff
@@ -46,43 +46,43 @@ export class DwSurface extends LitElement {
 
 
         /* STARTS: elevation styles based on elevation */
-        :host([elevation="0"]){
+        :host([elevation="0"]:not([noBoxShadow])){
```

Comment:
As per our standard practice, attribute names should always be dash-separated and all lower cases. So, change this to `no-box-shadow`. Please, don't forget to update the demo page too.